pkg/controllers/componentroutesecretsync/component_route_secret_sync_controller.go

```diff
@@ -0,0 +1,86 @@
+package componentroutesecretsync
```

Comment:
I would expect the observer disappear with this commit. Actually, the observer does not seem necessary with this commit it?

Reply:
This is not an observer, this is a controller that syncs a secret from the cluster-admin provided location to the known secret location.